lib/opal/cli_runners/chrome.js

```diff
@@ -91,7 +91,7 @@ CDP(options, function(client) {
     Runtime.exceptionThrown(function(exception) {
       var exceptionDetails = exception.exceptionDetails,
           properties = exceptionDetails.exception.preview.properties,
-          stackTrace = exceptionDetails.stackTrace.callFrames,
+          stackTrace = exceptionDetails.stackTrace ? exceptionDetails.stackTrace.callFrames : [],
```

Comment:
`exceptionDetails.stackTrace` can be undefined.
I should probably create a dedicated pull request with just this change, wdyt @elia ?

Reply:
Done :smile: